### DISCUSSION TITLE
Please add Caprice32 core

### DISCUSSION THREAD

**Discussion by deadfraggle** [OnionUI/Onion#546] [Ideas]
If possible.  It is the only core I could get to run certain CPC games. Thanks.

**Comment by Sichroteph** [comment]
I will ask DJM, our onion emulator expert

**Comment by freq-mod** [comment]
I second that. CrocoDS has many imperfections and reduced compatibility with some software. (straight up refused to load some larger .dsk files, crashing when loading some demos) Caprice32 used to have much better results than Croco, while being fairly lightweight (runs on v90!), so it could be added as an option.

**Reply by oherau** [reply]
Plus Caprice32 supports many extensions:
* .dsk (also in crocoDS)
* .sna (also in crocoDS)
* .tap
* .cdt
* .voc
* .m3u : very useful for multidisk roms
* .cpr
* .zip

The only drawback is support for .kcr (explained [here](https://github.com/redbug26/crocods-core/wiki/kcr))
